internal/protocol/core.go

```diff
@@ -76,9 +81,7 @@ func (m *Member) Attributes() map[string]string {
 }
 
 func (m *Member) String() string {
-	memberInfo := fmt.Sprintf("Member [%s]:%d - %s", m.Address().Host(), m.Address().Port(),
-		m.UUID(),
-	)
+	memberInfo := fmt.Sprintf("Member %s - %s", m.address.String(), m.UUID())
```

Comment:
`.String()` is unnecessary, `m.address` is enough.

Reply:
ok